movement.cpp

```diff
@@ -0,0 +1,435 @@
+{
+  front_left_motor_.changePwm(230.0);
+  front_right_motor_.changePwm(230.0);
+  back_left_motor_.changePwm(230.0);
+  back_right_motor_.changePwm(230.0);
+  front_left_motor_.forward();
+  front_right_motor_.backward();
+  back_left_motor_.backward();
+  back_right_motor_.forward();
+}
+
+void Movements::move135()
+{
+  front_left_motor_.changePwm(230.0);
+  front_right_motor_.changePwm(230.0);
+  back_left_motor_.changePwm(230.0);
+  back_right_motor_.changePwm(230.0);
+  front_left_motor_.stop();
+  front_right_motor_.backward();
+  back_left_motor_.backward();
+  back_right_motor_.stop();
+}
+
+void Movements::move180()
+{
+  front_left_motor_.changePwm(230.0);
+  front_right_motor_.changePwm(230.0);
+  back_left_motor_.changePwm(230.0);
+  back_right_motor_.changePwm(230.0);
+  front_left_motor_.backward();
+  front_right_motor_.backward();
+  back_left_motor_.backward();
+  back_right_motor_.backward();
+}
+
+void Movements::move225()
+{
+  front_left_motor_.changePwm(230.0);
+  front_right_motor_.changePwm(230.0);
+  back_left_motor_.changePwm(230.0);
+  back_right_motor_.changePwm(230.0);
+  front_left_motor_.backward();
+  front_right_motor_.stop();
+  back_left_motor_.stop();
+  back_right_motor_.backward();
+}
+
+void Movements::move270()
+{
+  front_left_motor_.changePwm(230.0);
+  front_right_motor_.changePwm(230.0);
+  back_left_motor_.changePwm(230.0);
+  back_right_motor_.changePwm(230.0);
+  front_left_motor_.backward();
+  front_right_motor_.forward();
+  back_left_motor_.forward();
+  back_right_motor_.backward();
+}
+
+void Movements::move315()
+{
+  front_left_motor_.changePwm(230.0);
+  front_right_motor_.changePwm(230.0);
+  back_left_motor_.changePwm(230.0);
+  back_right_motor_.changePwm(230.0);
+  front_left_motor_.stop();
+  front_right_motor_.forward();
+  back_left_motor_.forward();
+  back_right_motor_.stop();
+}
+
+void Movements::movement(int angle)
+{
+  switch (angle)
+  {
+  case 0: //Forward.
+    move0();
+    break;
+
+  case 45: //Diagonl front right.
+    move45();
+    break;
+
+  case 90: //Right.
+    move90();
+    break;
+
+  case 135: //Diagonal back right.
+    move135();
+    break;
+
+  case 180: //Backwards.
+    move180();
+    break;
+
+  case 225: //Diagonal back left.
+    move225();
+    break;
+
+  case 270: //Left.
+    move270();
+    break;
+
+  case 315: //Diagonal front left.
+    move315();
+    break;
+  }
+}
+
+void Movements::angleMovement(int angle)
+{
+  /* This funtcion allows the robot to move in the angle specified, the front of the robot should.
+  *  The function simulates a plane, where the front of the robot would the angle 0, it goes
+  *  in a clockwise motion, meaning that using a 90 degrees angle, would make the robot go to the right.
+  */
+  float pi = 3.141592;
+  int realAngle = angle + 45;
+  if (realAngle > 359)
+  {
+    realAngle -= 360;
+  }
+  float gradRadian = (realAngle * pi) / 180;
+  float pwm1 = 0, pwm2 = 0;
+  bool m1 = 0, m2 = 0;
+  float vc = 230;
```

Comment:
what does vc mean use names of variables more explicit, same for m1 and m2. Aplly this to comment to all your code.